packages/block-library/src/site-title/index.php

```diff
@@ -36,7 +36,7 @@ function render_block_core_site_title( $attributes ) {
 		if ( '_blank' === $attributes['linkTarget'] ) {
 			$link_attrs[] = 'target="_blank"';
 		}
-		$site_title = sprintf( '<a %1$s>%2$s</a>', implode( ' ', $link_attrs ), esc_html( $site_title ) );
+		$site_title = sprintf( '<a %1$s>%2$s</a>', implode( ' ', $link_attrs ), $site_title );
```

Comment:
Generated by Core function - can be trusted, maybe filtered.

Reply:
I can see that the "adminbar" escapes the site title.

![CleanShot 2022-02-07 at 16 25 36](https://user-images.githubusercontent.com/240569/152787814-f8128479-5a96-453e-92bf-576e051a140e.png)

Reply:
I've restored the $site_title escaping.